api/envoy/type/matcher/v3/regex.proto

```diff
@@ -45,6 +45,11 @@ message RegexMatcher {
     //
     // This field is deprecated; regexp validation should be performed on the management server
     // instead of being done by each individual client.
+    //
+    // .. note::
+    //
+    //  The program size is still checked against the runtime value.
```

Comment:
I think this note can be clarified a bit. What do you think of the following:
```suggestion
    //  Although this field is deprecated, the program size will still be checked against the *max_program_size* runtime value.
```

Reply:
Sure